package.json

```diff
@@ -2,8 +2,18 @@
   "name": "shx",
   "version": "0.0.1",
   "description": "Portable Shell Commands for Node",
-  "bin": { "shx" : "cli.js"},
+  "bin": {
+    "shx": "lib/cli.js"
+  },
   "scripts": {
+    "prebuild": "rimraf lib",
+    "build": "babel src -d lib",
+    "build:watch": "npm run build -- -w",
+    "dev": "npm run build:watch & npm run lint:watch",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint --silent -- --fix",
+    "lint:watch": "watch 'npm run lint --silent' src",
+    "postinstall": "npm run build",
```

Comment:
forgive me if i'm preaching to the choir here, for clarification:

- `rimraf` - cross platform `rm -rf`
- `--` in npm scripts is passing arguments from one npm script to the next
- `--silent` used in lint scripts to squelch `npm ERR` blocks (keeps lint errors clean on terminal)